graceful_controller_ros/src/graceful_controller_ros.cpp

```diff
@@ -376,37 +376,44 @@ class GracefulControllerROS : public nav_core::BaseLocalPlanner
 
         // Compute commands
         double vel_x, vel_th;
-        if (!controller_->approach(error.pose.position.x, error.pose.position.y, error_angle,
```

Comment:
so we are never using GracefulController::approach anymore?

Reply:
L398 - things are just moved around